### ISSUE TITLE
Feature Request: Add Alt Text to Images

### ISSUE THREAD

**Issue by dangero2000** [openrecall/openrecall#84]
I am a visually impaired screen reader user. I have a difficult time using Open Recall fully due to the lack of alt text. [This How To Geek page](https://www.howtogeek.com/821673/what-is-alt-text-why-you-should-use-it-and-how-to-write-it/) does a good job of describing alt text. In short, alt text is text that shows up in place of an image if it can't be rendered for some reason. It is also the text that gets read by a screen reader so that people who are blind or visually impaired can understand what's in an image.

#### Implementation Suggestion

As I understand it, Open Recall functions by using OCR on the screenshots that it takes so that you can search for the words present in the screenshot. Perhaps the alt text could be the detected text. For example, if Open Recall took a screenshot of a page title from a Duck Duck Go search page that says '5 Interesting Facts About Dogs', the image code would look something like this:
```html
'<img src="path/to/screenshots/14.jpg" alt="5 Interesting Facts About Dogs">'
```
This is just an example. I'm not exactly sure what the OR code looks like. Regardless, [this page](https://www.w3schools.com/TAGS/att_alt.asp) is where I went to get info on implementing alt text in images on an HTML page. I'm not a great coder, but I can try to implement this feature myself and submit a pull request if you don't have much time to add it in on your own.
